### PR TITLE
testing double rewards

### PR DESCRIPTION
**Pull request by 0xchain1**

### CODE REVIEW

**Comment by 0xchain1** [comment]
[x] Double rewards showing on manage page
Screen Shot 2021-12-07 at 11 11 15 PM

[x] Double rewards showing on claim modal
Screen Shot 2021-12-07 at 11 51 20 PM

[x] Double rewards showing on withdraw mod
Screen Shot 2021-12-07 at 11 51 14 PM
al



- [ ] Manage button does not show up for v2 farms on main page



the 2nd reward does not show if it's not a double reward farm

File: src/state/stake/user-farms.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ChainId, Token, JSBI, Pair, WETH, TokenAmount } from '@trisolaris/sdk'
-import { USDC, DAI, WNEAR, TRI} from '../../constants'
-import { useMasterChefContract, useMasterChefV2Contract, MASTERCHEF_ADDRESS_V1 } from './hooks-sushi'
+import { USDC, DAI, WNEAR, TRI, fAURORA2} from '../../constants'
```
Comment (0xchain1) [inline src/state/stake/user-farms.ts:2]: remove fAURORA and replace with AURORA

File: src/state/stake/user-farms.ts [hunk 2]
```diff
@@ -104,29 +104,31 @@ export function useSingleFarm(version: string): StakingTri[] {
         // check for account, if no account set to 0
         const userInfoPool = JSBI.BigInt(userStaked.result?.['amount'])
         const earnedRewardPool = JSBI.BigInt(rewardsPending.result?.[0])
-        const totalSupplyAvailable = JSBI.BigInt(pairTotalSupplyState.result?.[0])
+        const earnedComplexRewardPool = JSBI.BigInt(complexRewardPending.result?.rewardAmounts?.[0] ?? 0)
 
         const stakedAmount = new TokenAmount(pair.liquidityToken, JSBI.BigInt(userInfoPool))
         const earnedAmount = new TokenAmount(TRI[ChainId.AURORA], JSBI.BigInt(earnedRewardPool))
+        const earnedComplexAmount = new TokenAmount(fAURORA2[ChainId.AURORA], JSBI.BigInt(earnedComplexRewardPool))
```
Comment (0xchain1) [inline src/state/stake/user-farms.ts:111]: remove fAURORA and replace with AURORA

File: src/state/stake/stake-constants.ts [hunk 3]
```diff
@@ -214,7 +235,47 @@ const AURORA_POOLS: StakingTri[] = [
...
     chefVersion: ChefVersions.V1
-  }
+  },
+  {
+    ID: 7,
+    poolId: 0,
+    tokens: [AURORA[ChainId.AURORA], WETH[ChainId.AURORA]],
+    stakingRewardAddress: MASTERCHEF_ADDRESS_V2[ChainId.AURORA],
+    lpAddress: '0x5eeC60F348cB1D661E4A5122CF4638c7DB7A886e',
+    rewarderAddress: '0xA9961707292DD323F2DdE57996282c7De0e52d14',
```
Comment (0xchain1) [inline src/state/stake/stake-constants.ts:245]: replace with correct rewarder address

**Review by baboobhaiya — APPROVED** [review]

**Pull request merged**